map_frame: "map_frame"
robot_frame: "robot_frame"
map_topic: "map_topic"
goal_pose_sub_topic_name: "goal_pose_sub_topic_name"
global_path_pub_topic_name: "global_path_pub_topic_name"
model_cube_size: "0.25"
bound_min: "-10"
bound_max: "10"
